src/app/public/modules/data-manager/models/data-manager-column-picker-sort-strategy.ts

```diff
@@ -0,0 +1,4 @@
+export enum SkyDataManagerColumnPickerSortStrategy {
+  None = 'None',
+  SelectedThenAlphabetical = 'SelectedThenAlphabetical'
```

Comment:
We've used camelcase for previous string enums. `'none'`, `'selectedThenAlphabetical'`